src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,28 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private final int minAge = 18;
+    private final int minCountCharacters = 6;
```

Comment:
- make these constants
- think about name minCountCharacters - I can not understand what does in mean. Name should reflect the sence of the field